orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/cluster/FindImageFromClusterTask.groovy

```diff
@@ -229,8 +230,45 @@ class FindImageFromClusterTask extends AbstractCloudProviderAwareTask implements
       }
     }.flatten()
 
+    List<Artifact> artifacts = imageSummaries.collect { placement, summaries ->
```

Comment:
Is this meant to be an interim solution to make it appear like all stages emit artifacts rather than image summaries?

Reply:
Yes. The idea is to do this in two takes. This first PR to make artifacts available where we produce/find images. Then the next iteration will be to actually use these artifacts for deployments which will in turn make them even more available.